index.rst

```diff
@@ -25,131 +25,168 @@ What can we help you with today?
-
 ----------------
 
 Explore
 -------
 
 Learn about the Aiven platform
 
-.. panels::
-    :card: shadow
+.. grid:: 1 2 2 2
+
+    .. grid-item-card::
+        :shadow: md
+        :margin: 2 2 0 0
+        :link: docs/products/postgresql
 
-    |icon-postgres| **PostgreSQL®** Powerful relational database platform. We have the latest versions, and an excellent selection of extensions.
+        |icon-postgres| **PostgreSQL®** Powerful relational database platform. We have the latest versions, and an excellent selection of extensions.
 
-    +++
+        .. button-link:: docs/products/postgresql
+            :align: right
+            :color: primary
+            :outline:
 
-    .. link-button:: docs/products/postgresql/index
-        :type: ref
-        :text: Read more
-        :classes: stretched-link
+            Read more
+    
 
-    ---
+    .. grid-item-card::
+        :shadow: md
+        :margin: 2 2 0 0
+        :link: docs/products/m3db
 
-    |icon-m3db| **M3** Distributed time-series database for scalable solutions, with M3 Coordinator included, and M3 Aggregator also available.
+        |icon-m3db| **M3** Distributed time-series database for scalable solutions, with M3 Coordinator included, and M3 Aggregator also available.
 
-    +++
+        .. button-link:: docs/products/m3db
+            :align: right
+            :color: primary
+            :outline:
 
-    .. link-button:: docs/products/m3db/index
-        :type: ref
-        :text: Read more
-        :classes: stretched-link
+            Read more
+    
 
-    ---
+    .. grid-item-card::
+        :shadow: md
+        :margin: 2 2 0 0
+        :link: docs/products/kafka
 
-    |icon-kafka| **Apache Kafka®** Streaming data pipelines for the modern enterprise. Apache MirrorMaker2 and Kafka Connect also available.
+        |icon-kafka| **Apache Kafka®** Streaming data pipelines for the modern enterprise. Apache MirrorMaker2 and Kafka Connect also available.
 
 
-    +++
+        .. button-link:: docs/products/kafka
+            :align: right
+            :color: primary
+            :outline:
 
-    .. link-button:: docs/products/kafka/index
-        :type: ref
-        :text: Read more
-        :classes: stretched-link
+            Read more
+    
 
-    ---
+    .. grid-item-card::
+        :shadow: md
+        :margin: 2 2 0 0
+        :link: docs/products/flink
 
-    |icon-flink| **Apache Flink®** Framework for definining powerful transformations of batch and streaming data sets. :badge:`beta,cls=badge-secondary text-black badge-pill`
+        |icon-flink| **Apache Flink®** Framework for definining powerful transformations of batch and streaming data sets. :bdg-secondary:`beta`
 
-    +++
+        .. button-link:: docs/products/flink
+            :align: right
+            :color: primary
+            :outline:
 
-    .. link-button:: docs/products/flink/index
-        :type: ref
-        :text: Read more
-        :classes: stretched-link
-    ---
+            Read more
```

Comment:
What do you think if we remove "Read more" buttons completely? Now that with the new approach the whole panel became clickable, it is already gives a hint where the user will be redirected.

Reply:
<img width="841" alt="Screenshot 2022-07-03 at 21 45 00" src="https://user-images.githubusercontent.com/4600541/177055075-9010575f-feeb-4317-82a6-7cbcfe5acc19.png">

Reply:
Unless we must have that button to have a linkable panel. But in that case - would be better if the border color is not red.

Reply:
Thanks! This is really good feedback. We do need buttons (cards aren't usually a clickable UX element, and there's no visual clue, it's just a nice enhancement that you don't actually need to hit the button exactly). I agree that it looks a bit funky, and I might wait until we have #1037 with all its colour disasters in place before I finalise this.

Reply:
To be entirely correct, the button should be there, and the panel should _not_ be clickable since it's really a display thing, and the user might not expect it to be UI control. Anyway I've kept the clickable panel on the cards like this that have only a single button. Other cards have other content, such as a list of links, so having a single click destination on the panel makes no sense anyway (we probably should get some actual UX advice on this at some point, but right now I'm interested in upgrading the plugin).

I think it looks OK so I'll resolve and see if anyone else raises it still, then we can revisit.